### PR TITLE
acpica-tools: init at 20180209

### PR DESCRIPTION
**Pull request by tadfisher**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/system/acpica-tools/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  src = fetchurl {
+    url = "https://acpica.org/sites/acpica/files/acpica-unix-${version}.tar.gz";
+    sha256 = "1rpdfwa4vwnvyxdp9ygqjckmabc3s8kyg3jyq4n4f0rhr1zl4zy5";
+  };
+
+  NIX_CFLAGS_COMPILE = "-O3";
+
+  enableParallelBuilding = true;
+
+  buildFlags = stdenv.lib.concatStringsSep " " [
```
Comment (joachifm) [inline pkgs/tools/system/acpica-tools/default.nix]: Not hugely important, but is `concatStringsSep` really necessary? I'd expect it to end up being exported into the builder environment as a space separated string anyways.

File: pkgs/tools/system/acpica-tools/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "acpibin"
+    "acpidump"
+    "acpiexec"
+    "acpihelp"
+    "acpinames"
+    "acpixtract"
+  ];
+
+  buildInputs = [ bison flex ];
```
Comment (joachifm) [inline pkgs/tools/system/acpica-tools/default.nix]: Both of these can be treated as `nativeBuildInputs`, if they are used only for codegen during the build.

File: pkgs/tools/system/acpica-tools/default.nix (hunk 1 above)
Comment (tadfisher) [reply pkgs/tools/system/acpica-tools/default.nix]: I think you're right, and the Nixpkgs manual needs some clarification.

File: pkgs/tools/system/acpica-tools/default.nix (hunk 2 above)
Comment (tadfisher) [reply pkgs/tools/system/acpica-tools/default.nix]: Thanks, fixed.

**Comment by joachifm** [comment]
@GrahamcOfBorg build acpica-tools

**Comment by joachifm** [comment]
Thank you

**Pull request merged**